### DISCUSSION TITLE
Allocate Physical Cell Identifier (PCI) to BS

### DISCUSSION THREAD

**Discussion by sundar19** [stefanbschneider/mobile-env#27] [Q&A] [answered]
Hello,
I need to allocate PCI (using RL algorithms) to BS. I saw the example implementation of multi_agent.py in handlers. Is it possible to have more BS than the number of PCIs and use RL algorithms to allocate PCI to BS , considering collision and confusion free network?

**Comment by stefanbschneider** [comment, marked as answer]
Hi @sundar19 , as far as I remember, we do not explicitly model physical cell identifiers (PCIs). So you'd have to extend mobile-env to include PCI, which should be doable. You would also have to adjust the actions to do PCI allocation to BS rather than UE allocation to BS.

Overall, PCI to BS allocation is not supported out of the box with mobile-env. But it should be extendible to what you want. Maybe @stwerner97 can also help here if you have questions :)

**Comment by sundar19** [comment]
Thank you so much @stefanbschneider , I am figuring out how to extend **mobile-env** for that purpose. As of now I assigned numbers to all the BS instead of alphabets. I will try to deploy some RL techniques which can in turn handle allocating these assigned numbers (PCI) to BS. I will definitely keep you posted if I find any ways to add this feature.

**Comment by stweigand97** [comment]
Hi @sundar19! I am happy to answer any questions you may have concerning our implementation of ``mobile-env``.

**Comment by sundar19** [comment]
Thank you very much @stwerner97
